### ISSUE TITLE
Unearthed Arcana Skill & Racial feats

### ISSUE THREAD

**Issue by LordMorin** [Sklore/HL_DD_5e_Colab#647]
Is anyone already working on the new UA Skill feats:

http://media.wizards.com/2017/dnd/downloads/UA-SkillFeats.pdf

or Racial feats?

https://media.wizards.com/2017/dnd/downloads/RJSJC2017_04UASkillFeats_24v10.pdf

If not I'll create them for the Community pack. Otherwise I'll probably still create them for my own use. :)

**Comment by Sklore** [comment]
not that im aware of

**Comment by LordMorin** [comment]
KK, I'll go ahead and add them.

There's already a source for UA feats from the tool feats put out qa while back, so I'll use that.

**Comment by eqbourne** [comment]
total - noob here - but is the Warlock Ability - The Hexblade being added or worked on?

**Comment by LordMorin** [comment]
It's complete and can will be in the next Community pack update. You can download it early from this site if you want to play with it. I've done that myself. I really like the Hexblade.

**Comment by eqbourne** [comment]
Which file would it be in? Can you point in the right direction? Thanks

**Comment by LordMorin** [comment]
I never remember where to download the complete file. I always have to ask myself.

**Comment by Sklore** [comment]
HERE: https://github.com/Sklore/HL_DD_5e_Colab/archive/master.zip

**Comment by LordMorin** [comment]
I was feeling better this weekend so I started working on these feats again. I got the issue I was having with Proficiency and Double Proficiency resolved so the rest is easy mode so far in relation to scripting.

So far, so good.

**Comment by LordMorin** [comment]
The skill feats are almost done. Two of them are giving me issues though.

1) Perceptive. The script I use on all the other feats to give Proficiency or Double Proficiency doesn't work properly for elves who gain Proficiency with Perception through the Keen Senses racial ability. I'll look at it again today.
2) The Brawny feat works fine but I have to modify it slightly so that it stacks with the Goliath's Powerful Build ability and the Barbarianss Bear Totem ability. I need to find out if they do stack.

http://www.dndbeyond.com/forums/dungeons-dragons-discussion/rules-game-mechanics/2613-goliath-ability-and-feat-stacking

https://www.reddit.com/r/dndnext/comments/66efsb/do_you_think_goliathsfirbolgs_powerful_build/

Let me know if you want me to upload them as is for now (once I resolve the Perceptive feat/Ellllf Keen Sight issue).

**Comment by LordMorin** [comment]
Ok, I think I have all the bugs out of these feats (thanks in part to ShadowChemosh. Many thanks).

The only thing is that for some reason, GIT won't let me attach a .zip file containing my .user file. :(

It says that I can drag & drop, or select, any PNG, GIF, JPG, DOCX, PPTX, XLSX, TXT, PDF, or ZIP files, but when I try to attach a zip it rejects it. Any ideas?

**Comment by LordMorin** [comment]
I've started adding the Unearthed Arcana Racial feats.

**Comment by DarkHayZe** [comment]
Wow... WinRaR is still around? I highly recomend 7-zip. 

Why not just upload it directly to the repository?

**Comment by LordMorin** [comment]
Last time I tried to upload directly to the repository, I messed it up. I prefer to create standalone files and let true experts deal with the integration. When I create stuff for the 5E Community pack these days I try to follow its naming standards. The Pack already had a Source for Unearthed Arcana feats so I used that as Source.

I'll give 7-zip a try and see if it is accepted. WinRAR 5.40 is the latest version short of the 5.55 beta. It SHOULD work . . . <grumble>. :)

**Comment by DarkHayZe** [comment]
7-zip is amazing, it is 100% free and handles just about every file archive format

**Comment by LordMorin** [comment]
I downloaded and installed 7-zip. I created a zip with my .user file. When I tried to drag & drop it into a windows here, I get the message: 'We don’t support that file type. with a PNG, GIF, JPG, DOCX, PPTX, XLSX, TXT, PDF, or ZIP'.

<sigh>

**Comment by DarkHayZe** [comment]
Maybe it's your browser, did you update or install a new plug in?

**Comment by LordMorin** [comment]
I've tried Chrome with the same issue. I also tried disabling my Anti-virus in case it was causing an issue.

You did give me an idea though. It MIGHT be Ad=Blocker. I try it and find out.

**Comment by LordMorin** [comment]
Disabled ABP on github.com to no availe . . .

**Comment by DarkHayZe** [comment]
Any luck?

**Comment by LordMorin** [comment]
Not so far. Plus I am back to scratch on parts of my scripting. I have to
re-do all the script that grants Proficiency or Double Propficiency. The
problem is it sometimes works and sometimes doesn't.

I need to get it working properly before I worry about getting it uploaded.
:/

**Comment by DarkHayZe** [comment]
I was going to say, if you want to email it to me I can take a look / get it uploaded to you

**Comment by DarkHayZe** [comment]
I have a question... that you might be able to answer... I am having some pre-req issues with a file I am working on.... I have a couple feats that require you to be one of two out of three subraces, but I cannot get the pre-req to work for them. Either it blocks it completely or doesn't block it at all.

I thought since some of these new feats are racial, the answer might fall in there. Would you mind sending me the file you do have so I can take a look?

**Comment by LordMorin** [comment]
I was about to post a question to help. :)

**Comment by dungeonguru** [comment]
I uploaded a Racial Feats file with my last uploads, it might conflict with what you folks have. Not sure if you want to pull mine and see if it's in line with what you wanted or not?

**Comment by dungeonguru** [comment]
I merged my personal files (with updated 5C identifier) for the Skills for Feats. I'm going to close this ticket since some of these feats are overwritten by Xanathar's Racial Feats and we already pushed those out. The Brawny feat currently stacks with others like Bear totem and Firbolg, but this is something we'll revisit if we get official clarification - the UA feats are "not official" so I'm not sure if we'll get a good answer.
